content/departments/engineering/teams/dev-experience/newsletter.md

```diff
@@ -8,6 +8,48 @@ To learn more about components of Sourcegraph's developer experience, check out
 
 > NOTE: For authors, refer to [this guide](./processes.md#newsletter) for preparing a newsletter.
 
+## July xx, 2022
+
+Welcome to another iteration of the [Developer Experience newsletter](./newsletter.md)!
+As a reminder, you can check out previous iterations of the newsletter in the [newsletter archive](./newsletter.md).
+
+## Frontend news
+
+**esbuild for faster frontend builds**: Shoutout to @Nick Snyder who took it upon himself to get esbuild in a usable state for Sourcegraph. It doesn't work for everything yet but a few people have reported a markable improvement in sg startup time! You can enable it by running `DEV_WEB_BUILDER=esbuild sg start`.
```

Comment:
It would be nice to link [the documentation](https://docs.sourcegraph.com/dev/background-information/web/build#esbuild) we have for `esbuild` for people interested in getting more context.

Reply:
Awesome. I'll add it!